### ISSUE TITLE
Question About train cmu

### ISSUE THREAD

**Issue by andyqian2015** [ildoonet/tf-pose-estimation#47]
Hi @ildoonet，soory to intrupt you . I try to train the cmu model as what you said in the train.md.
I use the command:
```
python3 train.py --model=cmu --datapath={datapath} --batchsize=64 --lr=0.001 --modelpath={path-to-save}
```
I change the lr to 0.0001 as what you tell in the readme,batchsize set to 16（10 would get an err）,
when I run amost 8000 iter,the loss is still 1000 more or less. I just don't know what happen. can you give me some clue? thx

**Comment by ouceduxzk** [comment]
also the two figures in his https://github.com/ildoonet/tf-pose-estimation/blob/master/etcs/training.md does not make sense, where the x axis is not the # of iteration and it was truncated somehow

**Comment by elmuz** [comment]
> also the two figures in his https://github.com/ildoonet/tf-pose-estimation/blob/master/etcs/training.md does not make sense, where the x axis is not the # of iteration and it was truncated somehow

To me they make sense. They just far away from the results I got. Why you say that x axis isn't the # of iterations?

**Comment by ouceduxzk** [comment]
@elmuz  how is it possible to converge so fast, the original author claimed to be 440k iterations, this   one only 20k iterations

**Comment by elmuz** [comment]
Maybe it's because he could use massive batch sizes, couldn't it be?

**Comment by ouceduxzk** [comment]
@elmuz  yes, that's the reason, it just occured to me. Anyway, I am trying to run a few new experiments to see whether i can get the similar result

**Comment by tranorrepository** [comment]
I train with mobilenet, the loss is also quite high

**Comment by andyqian2015** [comment]
@tranorrepository  how much iter do you execute? I check in the code that the max iter is about 456.000k

**Comment by ouceduxzk** [comment]
I tried a different architecture, now the validation loss goes down to 422 with just 6k iterations

**Comment by andyqian2015** [comment]
@ouceduxzk ，what parameter do you choose:the batchsize and the lr is the same as the author?

**Comment by tranorrepository** [comment]
@andyqian2015 I trained 200K+ iterations

**Comment by ouceduxzk** [comment]
training with mobilenet now, the loss is high because the loss is calculated on 92x92 feature map instead of 46x46 in the mobilenet_thin case, so you would to expect the loss to be 4 times higher

**Comment by ouceduxzk** [comment]
another tip for those who are trying to find the best combo of architecture and hyperparams, the initial value of loss in the first few hundred steps are very indicative of your final result, the lower the initial value the better (roughly)
